src/sylveon/_config/config.py

```diff
@@ -0,0 +1,63 @@
+"""config.py
+Configuration file for FastAPI. Specifies the specific objects and data models used in our api
+"""
+
+import requests
+from enum import Enum
+from pydantic import BaseModel
+
+
+class SemesterModel(str, Enum):
+    spring = "spring"
+    summer = "summer"
+    fall = "fall"
+    winter = "winter"
+
+
+class SessionModel(BaseModel):
+    session: requests.Session
+
+
+class ConnectionModel(BaseModel):
+    session: SessionModel
+    logged_in: bool = False
+
+
+class TokenModel(BaseModel):
+    access_token: str
+
+
+class CourseModel(BaseModel):
+    course_id: str
+    course_short_name: str
+    course_name: str
+    course_semester: SemesterModel
+    course_year: int
+
+
+class UserModel(BaseModel):
+    user_id: str
```

Comment:
Does gradescope have user ids/usernames? Or is this just a placeholder for like the user's full name

Reply:
just acts as a placeholder for the user's full name/whatever identifier Gradescope will use to identify individual users